assets/css/_info-bubble.scss

```diff
@@ -181,18 +181,16 @@
 
   .non-variant {
     float: right;
-    margin-right: 8px;
-    margin-top: 5px;
-    margin-bottom: 8px;
-    width: 150px;
-    height: $info-bubble-button-size;
-    line-height: $info-bubble-button-size;
-
-    user-select: none;
-
-    white-space: nowrap;
-
     text-align: right;
+    margin-top: 9px;
+    margin-left: 8px;
+    margin-right: 7px;
+    min-height: 30px;
+    white-space: nowrap;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
```

Comment:
We don't need to prefix CSS - the `autoprefixer` module takes care of this for us.